README.md

```diff
@@ -1,5 +1,5 @@
-RISC-V Debug Specification
-==========================
+Debug Specification for RISC-V
```

Comment:
Should this be "External Debug Specification for RISC-V"?  That's what is being called within the doc.

Reply:
This document doesn't only describe external debug (in particular, action=0 causes trap-based debug where no external debugger is present).  So since we're renaming the document anyway, I'll re-raise [my point from last year](https://github.com/riscv/riscv-debug-spec/issues/448) that the document name should not have the word "external".  I've talked to at least a few people who thought that the only way to do debug on RISC-V is with an external debugger.

Reply:
Good point, no objection.  The name should be consistent throughout.

Reply:
I'll do that.